src/HashBus.Twitter.BackFill/App.cs

```diff
@@ -1,35 +1,54 @@
 ﻿namespace HashBus.Twitter.BackFill
 {
+    using System;
+    using System.Threading.Tasks;
     using HashBus.NServiceBusConfiguration;
     using HashBus.Twitter.Monitor.CatchUp.Commands;
     using NServiceBus;
     using NServiceBus.Persistence;
 
     class App
     {
-        public static void Run(
+        public static async Task Run(
             string nserviceBusConnectionString,
             string endpointName,
             string track,
             long tweetId)
         {
-            var busConfiguration = new BusConfiguration();
-            busConfiguration.EndpointName(endpointName);
-            busConfiguration.UseSerialization<JsonSerializer>();
-            busConfiguration.EnableInstallers();
-            busConfiguration.UsePersistence<NHibernatePersistence>().ConnectionString(nserviceBusConnectionString);
-            busConfiguration.ApplyMessageConventions();
+            var endpointConfiguration = new EndpointConfiguration(endpointName);
+            endpointConfiguration.UseSerialization<JsonSerializer>();
+            endpointConfiguration.EnableInstallers();
+            endpointConfiguration.UsePersistence<NHibernatePersistence>().ConnectionString(nserviceBusConnectionString);
+            endpointConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyErrorAndAuditQueueSettings();
 
-            using (var bus = Bus.Create(busConfiguration).Start())
+            var transportExtensions = endpointConfiguration.UseTransport<MsmqTransport>();
+            var routing = transportExtensions.Routing();
+            routing.RouteToEndpoint(
+                messageType: typeof(StartCatchUp),
+                destination: "HashBus.Twitter.Monitor.CatchUp");
+
+            var endpointInstance = await Endpoint.Start(endpointConfiguration)
+                .ConfigureAwait(false);
+            try
             {
+                Console.WriteLine("\r\n Bus created and configured; press any key to stop program\r\n");
+
                 var command = new StartCatchUp
                 {
                     EndpointName = endpointName,
                     Track = track,
                     TweetId = tweetId,
                 };
 
-                bus.Send(command);
+                await endpointInstance.Send(command).ConfigureAwait(false);
+
+                Console.ReadKey();

```

Comment:
There is no need for this pattern here. All this method should do is start the bus, send the command, stop the bus, and then exit. This is what the current code does on master.